pkg/ccl/changefeedccl/parquet.go

```diff
@@ -115,6 +118,7 @@ func newParquetWriterFromRow(
 		if opts, err = addParquetTestMetadata(row, encodingOpts, opts); err != nil {
 			return nil, err
 		}
+		opts = append(opts, parquet.WithMetadata(parquet.MakeReaderMetadata(schemaDef)))
```

Comment:
@jayshrivastava do you know if this change will cause any existing tests to fail - it adds an additional metadata column which indicates the type of event for any buildutil.CrdbTestBuild builds